.babelrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
The thinking here was for two reasons.  NPM keeps jacking up the formatting and because `babelrc` is a file specifically for this use.  As our package.json continues to get longer, it's nice to not have it be a place for a bunch of different kinds of configs.